Translation corrections (#315)
07th-mod/higurashi-console-arcs: Update/kake_037.txt

			NULL, "\"Don't be absurd, it was more than enough!", Line_WaitForInput);
	ModPlayVoiceLS(4, 11, "ps3/s20/11/vtk1_oishi1109", 256, TRUE);
	OutputLine(NULL, "　鹿嶋桜…でしたか？",

> 			NULL, " It was Kashimazakura... wasn't it?", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, " It was Kashima Sakura... wasn't it?", Line_WaitForInput);
```

			NULL, "\"Anyway, it's a shame.", Line_WaitForInput);
	ModPlayVoiceLS(4, 11, "ps3/s20/11/vtk1_oishi1114", 256, TRUE);
	OutputLine(NULL, "ぜひとも今夜は、その鹿嶋桜の酒蔵を南井さんに教えてもらいたかったのに。",

> 			NULL, " It would've been nice if Minai-san could've been here tonight to tell us all about that Kashimazakura brewery. ", GetGlobalFlag(GLinemodeSp));

Reviewer: ```suggestion
			NULL, " It would've been nice if Minai-san could've been here tonight to tell us all about that Kashima Sakura brewery. ", GetGlobalFlag(GLinemodeSp));
```

			NULL, "\"Now that you mention it, I did hear he was injured when he went in.", Line_WaitForInput);
	ModPlayVoiceLS(4, 11, "ps3/s20/11/vtk1_oishi1122", 256, TRUE);
	OutputLine(NULL, "でも、杖つきでも歩けるってことは大事がなさそうで、何よりじゃないですか」",

> 			NULL, " But it's not so serious that he'll have to walk with a cane, so it's all good, is it?\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " But it's not so serious that he'll have to walk with a cane, so it's all good, isn't it?\"", Line_Normal);
```

//r警察のお偉方か、それとも政治家か…誰が初めに言い出したのかは知らないが、現場無視にもほどがあるというものだ。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "警察のお偉方か、それとも政治家か…誰が初めに言い出したのかは知らないが、現場無視にもほどがあるというものだ。",

> 			NULL, "Were they police officers VIPs or politicians...? I don't know who said it first, but neglect of the people in the field had its limits.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Were they police VIPs or politicians...? I don't know who said it first, but neglect of the people in the field had its limits.", Line_Normal);
```
